src/Java/IO/ReadCML.java

```diff
@@ -0,0 +1,84 @@
+import vtk.vtkActor;
+import vtk.vtkNamedColors;
+import vtk.vtkNativeLibrary;
+import vtk.vtkRenderWindow;
+import vtk.vtkRenderWindowInteractor;
+import vtk.vtkRenderer;
+import vtk.vtkCMLMoleculeReader;
+import vtk.vtkMoleculeMapper;
+
+public class ReadCML 
+{
+  // -----------------------------------------------------------------
+  // Load VTK library and print which library was not properly loaded
+  static 
+  {
+    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+    {
+      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+      {
+        if (!lib.IsLoaded()) 
+        {
+          System.out.println(lib.GetLibraryName() + " not loaded");
+        }
+      }
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+  
+
+  public static void main(String args[]) 
+  {
+    //parse command line arguments
+    if (args.length != 1) 
+    {
+      System.err.println("Usage: java -classpath ... Filename(.cml) e.g porphyrin.cml");
+      return;
+    }
+    String inputFilename = args[0];
+    
+    vtkNamedColors colors = new vtkNamedColors();
+
+    //For Actor Color
+    double actorColor[] = new double[4];
+    //Renderer Background Color
+    double Bgcolor[] = new double[4];
+
+    colors.GetColor("Tomato", actorColor);
```

Comment:
There is no need for actorColor, since `molmapper.UseBallAndStickSettings()` automatically sets the colors of the bonds and the carbon and nitrogen atoms.

Reply:
Have removed actorColor